### PR TITLE
Remove unused feature toggle configuration

### PR DESCRIPTION
**Pull request by mkllnk**
Removing unused functionality. Partnered with https://github.com/openfoodfoundation/openfoodnetwork/pull/7806.

This will create a merge conflict with https://github.com/openfoodfoundation/ofn-install/pull/751.

### CODE REVIEW

File: roles/app/templates/application.yml.j2 [hunk 1]
```diff
@@ -88,9 +88,3 @@ GEOCODER_SERVICE: {{ geocoder_service }}
 {% if geocoder_timeout is defined %}
 GEOCODER_TIMEOUT: {{ geocoder_timeout }}
 {% endif %}
-
-{{ custom_application_env_vars | default('') }}
```
Comment (Matt-Yorkley) [inline roles/app/templates/application.yml.j2:92]: This is still used in some cases.

File: inventory/host_vars/_example.com/config.yml [hunk 2]
```diff
@@ -26,9 +26,6 @@ mail_domain: example.com
 #
 #custom_hba_entries:
 #  - { type: hostssl, database: "{{ db }}", user: zapier, address: '54.86.9.50/32', auth_method: md5 }
-#
-#custom_application_env_vars: |
-#  OFN_FEATURE_CONNECT_AND_LEARN: "true"
```
Comment (Matt-Yorkley) [inline inventory/host_vars/_example.com/config.yml:31]: And we could keep an example here.

File: roles/app/templates/application.yml.j2 (hunk 1 above)
Comment (mkllnk) [reply roles/app/templates/application.yml.j2:92]: Is that in secrets? I couldn't find any use in ofn-install.

Comment (Matt-Yorkley) [reply roles/app/templates/application.yml.j2:92]: Yep :+1:

**Comment by sigmundpetersen** [comment]
What to do on this one @mkllnk ?
Close or continue?

**Review by jibees — APPROVED** [review]
I didn't know it exists in `ofn-install` ;)
I re triggered a launch, since the CI failed. I suspect this occurs sometimes, and it's not related to this PR in particular.

**Comment by mkllnk** [comment]
> I re triggered a launch, since the CI failed. I suspect this occurs sometimes, and it's not related to this PR in particular.

No, the build has been completely broken for a while. No need to re-trigger. It always fails unless we fix it.

**Pull request merged**